sirepo/template/radia_util.py

```diff
@@ -193,6 +193,11 @@ def build_container(g_ids):
     return radia.ObjCnt(g_ids)
 
 
+def build_racetrack(center=[0, 0, 0], size=[1, 1, 1], radii=[1, 2], sides=[1, 1], height=1, axis='x', calc='a', num_segs=3, curr_density=-1.0):
```

Comment:
I think it would be easier to just pass the object in and have this method pull out the bits it needs.

Reply:
This may be misguided, but I want to keep ```radia_util``` as agnostic as possible as to the details of sirepo objects. This is partly so that the ```geometryReport.py``` generated from the template can remain "pure" for use in notebooks (there is work to do in this regard)

Reply:
Ok, I think we should add this to the SW agenda too. I think in other circumstances (ex rsopt) we lean heavily on using sirepo objects outside of sirepo.